core/storage/classifier/gae_models_test.py

```diff
@@ -49,3 +49,35 @@ def test_create_new_classifier_runs_successfully(self):
         self.assertEqual(classifier.algorithm_id, 'LDAStringClassifier')
         self.assertEqual(classifier.cached_classifier_data, {'alpha': 1.0})
         self.assertEqual(classifier.data_schema_version, 1)
+
+
+class TrainClassifierJobModelUnitTests(test_utils.GenericTestBase):
+    """Test the TrainClassifierJobModel class."""
+
+    def setUp(self):
+        super(TrainClassifierJobModelUnitTests, self).setUp()
+        classifier_models.TrainClassifierJobModel.create(
+            'LDAStringClassifier', 'exp_id1', 1, 'state_name1', 'PENDING',
+            [{'answer_group_index': 1, 'answers': ["a1", "a2"]}])
```

Comment:
Ah, switching back and forth between Java and Python does that to you. Thanks! :D